packages/common/package.json

```diff
@@ -52,6 +52,7 @@
     "@configu-integrations/toml": "workspace:*",
     "@configu-integrations/configu": "workspace:*",
     "@configu-integrations/json-file": "workspace:*",
+    "fast-glob": "^3.3.2",
```

Comment:
@jamesL92 can you please attach an npm trends or similar comparison between glob libs 🙏🏽  why did you pick `fast-glob`?

Reply:
Picked `fast-glob` as it _was_ already being used in the repository before (see [here](https://github.com/configu/configu/blob/8eca84c9a30e1e15599251b19440df7e2553d0df/packages/cli/package.json#L111), specifically was used quite a bit for the old [find](https://github.com/configu/configu/blob/8eca84c9a30e1e15599251b19440df7e2553d0df/packages/cli/src/commands/find.ts) command prior to migration).

Can see it's no longer being used by the CLI since the migration to `clipanion`, although I guess that's mainly because the `find` command hasn't been ported yet. As such, I'm happy to take a look into a comparison between different libraries if you'd like :+1: